release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+[[release-notes-ocp-3-7-release-notes]]
+= {product-title} 3.7 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Red Hat {product-title} is a Platform as a Service (PaaS) that provides
+developers and IT organizations with a cloud application platform for deploying
+new applications on secure, scalable resources with minimal configuration and
+management overhead. {product-title} supports a wide selection of
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Google Kubernetes, {product-title}
+provides a secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while providing integrated application runtimes
+and libraries. {product-title} brings the OpenShift PaaS platform to customer
+data centers, enabling organizations to implement a private PaaS that meets
+security, privacy, compliance, and governance requirements.
+
+[[ocp-36-about-this-release]]
+== About This Release
+
+Red Hat {product-title} version 3.7
+(link:https://access.redhat.com/errata/RHBA-2017:3188[RHSA-2017:3188]) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.7.0-rc.0[OpenShift
+Origin 3.7]. New features, changes, bug fixes, and known issues that pertain to
+{product-title} 3.7 are included in this topic.
+
+{product-title} 3.7 is supported on RHEL 7.3, 7.4, and Atomic Host 7.4.2 and
+newer with the latest packages from Extras, including Docker 1.12.
+
+For initial installations, see the
+xref:../install_config/install/planning.adoc#install-config-install-planning[Installing
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+To upgrade to this release from a previous version, see the
+xref:../install_config/upgrading/index.adoc#install-config-upgrading-index[Upgrading
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+[[ocp-36-new-features-and-enhancements]]
+== New Features and Enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[[ocp-37-container-orchestration]]
+=== Container Orchestration
+
+[[ocp-37kubernestes-upstream]]
+==== Kubernetes Upstream
+
+Many core features Google announced in June for Kubernetes 1.7 were the result
+of OpenShift engineering. Red Hat continues to influence the product in the
+areas of storage, networking, resource management, authentication and
+authorization, multi-tenancy, security, service deployments, templating, and
+controller functionality.
+
+[[ocp-37-crio]]
+==== CRI-O (Technology Preview)
+
+CRI-O v1.0 (currently in xref:ocp-37-technology-preview[Technology Preview]) is a
```

Comment:
- This should link to the docs on how to install it or deploy/upgrade a cluster so that you're using this!
- Missing a note (like what we had in 3.6): This feature is currently in Technology Preview and not for production workloads.